### ISSUE TITLE
Initialize PIP faster

### ISSUE THREAD

**Issue by orlp** [dekuScrub731/picture-in-picture#6]
Right now it's only checked once per game tick whether or not Runelite is minimized, making the PIP
feel sluggish when alt tabbing. I'd suggest checking on every client tick instead.

**Comment by dekuScrub731** [comment]
When moving the initialization to `onClientTick`, I've been getting multiple PIP windows (fires too fast). Taking a closer look to figure out a solution.

**Comment by dekuScrub731** [comment]
This change will be reflected once v1.4.1 is merged into the RL plugin hub
